src/EA.Weee.Web/Areas/Admin/Controllers/AatfController.cs

```diff
@@ -87,7 +83,11 @@ public async Task<ActionResult> ManageAatfs(ManageAatfsViewModel viewModel)
             {
                 using (var client = apiClient())
                 {
-                    viewModel.AatfDataList = await GetAatfs(viewModel.Filter);
+                    viewModel = new ManageAatfsViewModel
+                    {
+                        AatfDataList = await GetAatfs(viewModel.Filter),
```

Comment:
Dunno if its me being blind again but cant see a test that sets the aatfDataList property of the viewmodel

Reply:
Um test ManageAatfsPost_ModelError_GetAatfsMustBeRun does a check that the call to get aatfs is called.

Reply:
I think we are ok, seems covered elsewhere... one of those days it seems